packages/shared/src/components/cards/SharedPostCardHeader.tsx

```diff
@@ -30,15 +30,19 @@ export const SharedPostCardHeader = ({
           size="xsmall"
         />
       </div>
-      <div className="flex flex-col flex-grow typo-footnote">
-        <span className="font-bold">{author.name}</span>
+      <div className="flex flex-col flex-1 flex-grow ml-2 typo-footnote">
+        <span className="font-bold line-clamp-2">{author.name}</span>
         <PostMetadata
-          className="text-theme-label-secondary"
+          className="line-clamp-1 text-theme-label-secondary !flex"
```

Comment:
No way! I never knew the usage of `!` prefix. Wow, I am learning a lot today 🤣

Reply:
Haha. Yeah can be useful but should not be overused 😃